### DISCUSSION TITLE
Ender 3 4.2.7 - BL Touch  - Back left side of bed low

### DISCUSSION THREAD

**Discussion by floydian68** [Jyers/Marlin#1573] [Q&A]
Issue: BL Touch prints well front to middle of bed.  Rear of bed (specifically left side) prints poorly.

Belts adjusted correctly
No binding detected
Bed level pre-BL Touch activation

Ideas?

**Comment by MrBanana2000** [comment]
I am having the same problem, did you ever find a solution? Thanks

**Comment by K2Kevin** [comment]
I had a similar issue, when doing mesh leveling, no matter how level the far left corner was, the values seemed off.   Because it was so far to the back corner of the print bed, the gantry wheel was pressing against the cable cover back there.  Re-leveled my bed, starting at a higher point (less compressed springs to begin with).  Problem resolved!

**Comment by Gamma36** [comment]
Backup your firmware. Then install UBL 10x10 firmware and create a mesh, Take a photo of the Mesh Viewer with values and upload it here.

It may then become obvious where your bed is twisted.

-Paul
